src/proto/grpc/testing/messages.proto

```diff
@@ -78,10 +77,13 @@ message SimpleRequest {
   bool fill_oauth_scope = 5;
 
   // Whether to request the server to compress the response.
-  bool request_compressed_response = 6;
+  bool response_compressed = 6;
 
   // Whether server should return a given status
   EchoStatus response_status = 7;
+
+  // Whether the server should expect this request to be compressed.
+  bool expect_compressed = 8;
```

Comment:
Didn't we say that this field should be nullable to make sure we don't change behavior of other interop tests?

https://github.com/google/protobuf/blob/814685ca2cd9280ca401e1842fd6311440921a0a/src/google/protobuf/wrappers.proto#L100